js/assessmentHelpers/calculateTooLongSentences.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Calculates the score based on the percentage of too long sentences.
+ * @param {number} percentage The percentage of too long sentences
+ * @returns {number} The score
+ */
+module.exports = function( percentage ) {
```

Comment:
Would prefer something like `sentenceLengthScore` or `sentenceLengthPercentageScore` instead of `calculateTooLongSentences`